src/Executables/Game/World/Entities/PlayerEntity.cs

```diff
@@ -122,8 +123,8 @@ public EAntiFlags AntiFlagGender
             _world = world;
             _logger = logger;
             _scope = serviceProvider.CreateScope();
-            var itemRepository = _scope.ServiceProvider.GetRequiredService<IItemRepository>();
-            Inventory = new Inventory(itemManager, _cacheManager, _logger, itemRepository, player.Id,
+            _itemRepository = _scope.ServiceProvider.GetRequiredService<IItemRepository>();
+            Inventory = new Inventory(itemManager, _cacheManager, _logger, _itemRepository, player.Id,
```

Comment:
Didn't investigate why the ItemRepository injected to the Inventory like this.

Reply:
Item interaction (UseItem, DropItem, etc) with this change causes EF Core exceptions.
I'm sad but i would recommend to revert this PR (i spend ~2 hours to fix it, but couldn't fully).

Reply:
@PoLaKoSz what are the exact errors?